React/Views/RCTWrapperViewController.m

```diff
@@ -133,4 +140,18 @@ - (void)didMoveToParentViewController:(UIViewController *)parent
   }
 }
 
+- (UIImageView *)findHairlineImageViewUnder:(UIView *)view {
+    if ([view isKindOfClass:UIImageView.class] && view.bounds.size.height <= 1.0) {
```

Comment:
Unfortunately no, this way preserves transparency. Other easier ways quickly mess up the transparency issue.